templates/membership/_mandates_script.html.twig

```diff
@@ -3,12 +3,12 @@
         var formMandates = $('#form__mandates');
         var formElected = $('#{{ form.elected.vars.id }}');
         var selectMandates = $('#{{ form.mandates.vars.id }}');
+        selectMandates.attr('required', {{ form.mandates.vars.required }});
```

Comment:
it's not correct, because now this field is always optional, but it should be required when `elected` checkbox is checked.

I think you can replace:

```js
selectMandates.attr('required', isChecked);
```
by
```js
selectMandates.prop('required', isChecked);
```